client/jetpack-connect/style.scss

```diff
@@ -656,3 +656,25 @@
 .jetpack-connect__navigation {
 	text-align: center;
 }
+
+.jetpack-connect__plan-info {
+	margin-top: 35px;
+	text-align: center;
+	width: 970px;
+}
+
+.jetpack-connect__plan-info-illustration {
+	margin: 30px 0 10px;
```

Comment:
Doesn't feel right to have this top space defined here; do we want to add it as a `padding-top` of the parent, so it will be there if we move the illustration or prepend something before it?

Reply:
Upon another visual check, we do not seem to need it at all. The card looks as-good more compact 👍